src/pug/sidebar.pug

```diff
@@ -1,23 +1,17 @@
+- var i = 0;
 script
     | console.warn('Deprecation warning: Please update your usage of Chrome.')
 script(src='./static/chrome/js/chrome.js')
 aside.pf-c-page__sidebar.pf-l-page__sidebar#ins-c-sidebar
     nav.pf-c-nav(aria-label='Insights Global Navigation')
         ul.pf-c-nav__list
-            li.pf-c-nav__item
-                a(class='pf-c-nav__link')
-                    div.ins-c-skeleton.ins-c-skeleton__lg &nbsp;
-            li.pf-c-nav__item
-                a(class='pf-c-nav__link')
-                    div.ins-c-skeleton.ins-c-skeleton__lg &nbsp;
-            li.pf-c-nav__item
-                a(class='pf-c-nav__link')
-                    div.ins-c-skeleton.ins-c-skeleton__lg &nbsp;
-            li.pf-c-nav__item
-                a(class='pf-c-nav__link')
-                    div.ins-c-skeleton.ins-c-skeleton__lg &nbsp;
+            while i < 4
+                li.pf-c-nav__item
+                    a.pf-c-nav__link.ins-c-skeleton__link
+                        .ins-c-skeleton.ins-c-skeleton__lg &nbsp;
+                        - i++ss
```

Comment:
`- i++ss` ?

Reply:
Sometimes my new keyboard likes to not pick up the ctrl input sometimes and I end up with things like this ugh